### PR TITLE
Added favicon

### LINKED ISSUE #12: Change favicon on tab
### What is it?
A favicon is the icon that shows up on each tab in your browser. Like the Octocat on Github's!

### Summary
Ask Harsha for a PNG version of PantherBot logo. Size down the image to around 120x120 pixels. Then make it the favicon for the website. A simple Google search like `rails 5 favicon` should explain how to do it.

### PR DESCRIPTION
**Pull request by Litagano-M**
Closes #12

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
     <%= stylesheet_link_tag    'style.css' %>
     <%= stylesheet_link_tag    'mobile.css' %>
+
+    <%= favicon_link_tag       'favicon.ico' %>
```
Comment (ferrerluis) [inline app/views/layouts/application.html.erb:23]: I saw this file. Could you please talk to Harsha and get Pbot only without the blue square background and without the shadow? Then replace the `favicon.ico` file with that one

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,6 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+# Ignore IDE config files
```
Comment (ferrerluis) [inline .gitignore:28]: Submit this in a separate commit pls

**Review by ferrerluis — CHANGES_REQUESTED** [review]
Some very small details. Good job so far! @Litagano-M

File: .gitignore (hunk 2 above)
Comment (Litagano-M) [reply .gitignore:28]: Will do!

**Comment by Litagano-M** [comment]
Changed.

**Pull request closed without merging**